### ISSUE TITLE
Add tooltip to explain meaning of each attribute/field of rating

### ISSUE THREAD

**Issue by carpenterd777** [Peer-Stevens/peer#278]
It would be nice if users could know the intended meaning of each attribute without having to guess what the interpretation means. This should be done using a tooltip that is screen-reader compatible and has text.
